Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocListControlObjectTest.cs

```diff
@@ -894,7 +1054,7 @@ public void TestGetCellMatchesExactly ()
               .With.Message.EqualTo(
                   AssertionExceptionUtility.CreateControlMissingException(
                           Driver,
-                          "Unable to find css: .bocListTable .bocListTableBody .bocListDataRow .bocListDataCell[data-boclist-cell-index='7'] span[data-boclist-cell-contents='EO']")
+                          "Unable to find css: .bocListTable .bocListTableBody .bocListDataRow .bocListDataCell[data-boclist-cell-index='7'] *[data-boclist-cell-contents='EO']")
```

Comment:
if we change the selector (see line 238 in the BocListControlObject), we'll need to update the exception message. This comment is just a note to anticipate the red test.